Readme.md

```diff
@@ -38,6 +38,11 @@ app.use(logger())
   Recommended that you `.use()` this middleware near the top
   to "wrap" all subsequent middleware.
 
+## Options
+
+* `logger` - an object implementing, at least, a `log` function with variadic arguments. (default `console`)
```

Comment:
Exactly. The point is to affect the current implementation as little as possible, while making customization also basic. You could just pass `{ log: function () { /* handle arguments */ } }` without specifying the `level` option.

I believe this is the best of both worlds.